docs/Advanced Configuration/System and Security.md

```diff
@@ -23,32 +23,34 @@ system:
   googlevisibility: false # 'true' to allow Google visibility (via robots.txt), 'false' to disallow
 ```
 
-## Local 
-If running Java directly, you can also pass these as properties using `-D` arguments such as:
-```
-java -jar Stirling-PDF.jar -DAPP_HOME_NAME="New Application Name"
```

Comment:
Can still include the -D part here just change the var name, its still a valid way of adding SECURITY_ENABLELOGIN etc

Reply:
Ah lol, sure thing